tests/flow/test_list.py

```diff
@@ -7,7 +7,48 @@
 redis_graph = None
 GRAPH_ID = "G"
 
+# tests the GRAPH.LIST command
+class testGraphList(FlowTestsBase):
+    def __init__(self):
+        self.env = Env(decodeResponses=True)
 
+    def create_graph(self, graph_name, con):
+        con.execute_command("GRAPH.EXPLAIN", graph_name, "RETURN 1")
+
+    def test_graph_list(self):
+        # no graphs, expecting an empty array
+        con = self.env.getConnection()
+        graphs = con.execute_command("GRAPH.LIST")
+        self.env.assertEquals(graphs, [])
+
+        # create graph key "G"
+        self.create_graph("G", con)
+        graphs = con.execute_command("GRAPH.LIST")
+        self.env.assertEquals(graphs, ["G"])
+
+        # create a second graph key "X"
+        self.create_graph("X", con)
+        graphs = con.execute_command("GRAPH.LIST")
+        graphs.sort()
+        self.env.assertEquals(graphs, ["G", "X"])
+
+        # create a string key "str", graph list shouldn't be effected
+        con.set("str", "some string")
+        graphs = con.execute_command("GRAPH.LIST")
+        graphs.sort()
+        self.env.assertEquals(graphs, ["G", "X"])
+
+        # delete graph key "G"
+        con.delete("G")
+        graphs = con.execute_command("GRAPH.LIST")
+        self.env.assertEquals(graphs, ["X"])
+
+        # delete graph key "X", no graph keys in the keyspace
+        con.execute_command("GRAPH.DELETE", "X")
+        graphs = con.execute_command("GRAPH.LIST")
+        self.env.assertEquals(graphs, [])
```

Comment:
It might be worth adding a test for renamed graphs (which is handled properly).
```
127.0.0.1:6379> GRAPH.QUERY G "CREATE (:L)"
1) 1) "Labels added: 1"
   2) "Nodes created: 1"
   3) "Cached execution: 0"
   4) "Query internal execution time: 1.286539 milliseconds"
127.0.0.1:6379> RENAME G NEW
OK
127.0.0.1:6379> GRAPH.LIST
1) NEW
```